how_to_do_things_safely_in_bash.md

```diff
@@ -195,6 +195,13 @@ Caution: Builtins like `local` and `export` are also commands, so this is still
 
 ShellCheck warns only about special commands like `local` in this case.
 
+If you still want to use `local`, this is correct:
```

Comment:
Sounds like a discouragement. We must not discourage such a fine thing.

How about: To use `local`, separate the declaration from the assignment: